dashboard/test/models/plc/enrollment_task_assignment_test.rb

```diff
@@ -24,16 +25,16 @@ class Plc::EnrollmentTaskAssignmentTest < ActiveSupport::TestCase
     task_assignments.first.complete_assignment!
     assert_equal 'completed', task_assignments.first.status
     unit_enrollment.reload
-    assert_not_equal 'completed ', unit_enrollment.status
+    assert_equal Plc::EnrollmentUnitAssignment::IN_PROGRESS, unit_enrollment.status
```

Comment:
nit: just to make sure the state is as expected after setup, assert status == START_BLOCKED before calling complete_assignment!